anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,191 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from anime_downloader.const import get_random_header
+from selenium.webdriver.common.by import By
+from urllib.parse import urlencode
+from urllib.parse import urlsplit
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from logging import exception
+from sys import platform
+import requests
+import os
+import click
+import time
+import json
+
+
+def get_data_dir(): #gets the folder directory selescrape will store data, such as cookies or browser extensions and logs.
+    APP_NAME = 'anime downloader'
+    return os.path.join(click.get_app_dir(APP_NAME), 'data')
+
+
+
+def open_config(): #as the name suggests, it reads the config
+    try:
+        APP_NAME = 'anime downloader'
+        with open(os.path.join(click.get_app_dir(APP_NAME), 'config.json'), 'r') as json_file:
```

Comment:
I still don't understand why you're not using `anime_downloader.config.Config`?

Reply:
Pretty sure it creates circular import

Reply:
you mean to import it? if yes, it will be a circular import